modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,122 @@
+// module included in logging-6-0-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-0_{context}"]
+= Logging 6.0.0
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:6693[OpenShift Logging Bug Fix Release 6.0.0]
+
+[id="logging-release-notes-6-0-0-compatibility-support-matrix"]
+= Compatibility and support matrix
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The components or features in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] status are experimental and are not intended for production use.
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
+====
+
+.Components support matrix
+[options="header"]
+|===
+
+| {logging-uc} Version 6+| Component Version
+
+| Operator | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+
+|6.0 | 0.4 (GA) | v1.1 (GA) | v3.1.0 (GA) | v0.1 (GA) | v0.1 (GA) | v0.37.1 (GA)
+
+|===
+
+[id="logging-release-notes-6-0-0-removal-notice"]
+== Removal notice
+
+In Logging 6.0, the support is removed for:
+
+* Red Hat managed Elasticsearch log storage solution. For example, elasticsearch-operator
+* Red Hat managed log visualization using Kibana and Console Log View Plugin
```

Comment:
Support still exists for the console log view plugin.  ONLY kibana was removed

Reply:
Since it was mentioned in the release notes text section, it';s mentioned here. I'll remove console view plugin.